gymnasium/envs/classic_control/cartpole.py

```diff
@@ -58,7 +58,7 @@ class CartPoleEnv(gym.Env[np.ndarray, Union[int, np.ndarray]]):
     ## Rewards
 
     Since the goal is to keep the pole upright for as long as possible, a reward of `+1` for every step taken,
-    including the termination step, is allotted. The threshold for rewards is 475 for v1.
+    including the termination step, is allotted. The threshold for rewards is 500 for v1.
```

Comment:
Thanks for fixing this, could we have a note for v0 as well, that the maximum reward is 200

Reply:
Done! I've updated the branch